---a/net/minecraft/item/EnumDyeColor.java
+++b/net/minecraft/item/EnumDyeColor.java
@@ -4,8 +4,7 @@
 import net.minecraft.util.EnumChatFormatting;
 import net.minecraft.util.IStringSerializable;
 
-public enum EnumDyeColor implements IStringSerializable
-{
+public enum EnumDyeColor implements IStringSerializable {
     WHITE(0, 15, "white", "white", MapColor.snowColor, EnumChatFormatting.WHITE),
     ORANGE(1, 14, "orange", "orange", MapColor.adobeColor, EnumChatFormatting.GOLD),
     MAGENTA(2, 13, "magenta", "magenta", MapColor.magentaColor, EnumChatFormatting.AQUA),
@@ -33,8 +33,7 @@
 
     private static final String __OBFID = "CL_00002180";
 
-    private EnumDyeColor(int meta, int dyeDamage, String name, String unlocalizedName, MapColor mapColorIn, EnumChatFormatting chatColor)
-    {
+    private EnumDyeColor(int meta, int dyeDamage, String name, String unlocalizedName, MapColor mapColorIn, EnumChatFormatting chatColor) {
         this.meta = meta;
         this.dyeDamage = dyeDamage;
         this.name = name;
@@ -43,67 +43,54 @@
         this.chatColor = chatColor;
     }
 
-    public int getMetadata()
-    {
+    public int getMetadata() {
         return this.meta;
     }
 
-    public int getDyeDamage()
-    {
+    public int getDyeDamage() {
         return this.dyeDamage;
     }
 
-    public String getUnlocalizedName()
-    {
+    public String getUnlocalizedName() {
         return this.unlocalizedName;
     }
 
-    public MapColor getMapColor()
-    {
+    public MapColor getMapColor() {
         return this.mapColor;
     }
 
-    public static EnumDyeColor byDyeDamage(int damage)
-    {
-        if (damage < 0 || damage >= DYE_DMG_LOOKUP.length)
-        {
+    public static EnumDyeColor byDyeDamage(int damage) {
+        if (damage < 0 || damage >= DYE_DMG_LOOKUP.length) {
             damage = 0;
         }
 
         return DYE_DMG_LOOKUP[damage];
     }
 
-    public static EnumDyeColor byMetadata(int meta)
-    {
-        if (meta < 0 || meta >= META_LOOKUP.length)
-        {
+    public static EnumDyeColor byMetadata(int meta) {
+        if (meta < 0 || meta >= META_LOOKUP.length) {
             meta = 0;
         }
 
         return META_LOOKUP[meta];
     }
 
-    public String toString()
-    {
+    public String toString() {
         return this.unlocalizedName;
     }
 
-    public String getName()
-    {
+    public String getName() {
         return this.name;
     }
 
-    static
-    {
+    static {
         EnumDyeColor[] var0 = values();
         int var1 = var0.length;
 
-        for (int var2 = 0; var2 < var1; ++var2)
-        {
+        for (int var2 = 0; var2 < var1; ++var2) {
             EnumDyeColor var3 = var0[var2];
             META_LOOKUP[var3.getMetadata()] = var3;
             DYE_DMG_LOOKUP[var3.getDyeDamage()] = var3;
         }
     }
-}
+}
